src/components/search-members/search-members.module.scss

```diff
@@ -0,0 +1,11 @@
+.searchContainer {
+  display: flex;
+  align-items: center;
+  padding: 6px;
```

Comment:
Can we look into `rem` here please instead of `px`

Reply:
sure